requirements_dev.txt

```diff
@@ -0,0 +1 @@
+pre-commit==3.5.0
```

Comment:
Yeah it's precisely because we don't have a `pyproject.toml` which is why I had this as temporary.

I can submit another PR to port to `pyproject.toml` then, and remove this file?

Reply:
That'd be great, thanks!